virtio-devices/src/watchdog.rs

```diff
@@ -0,0 +1,429 @@
+// Copyright 2017 The Chromium OS Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
```

Comment:
Did you actually port this code from crosvm? Otherwise you should at least add a portion of copyright for Intel.